libs/liblz4/Config.in

```diff
@@ -0,0 +1,11 @@
+menu "Configuration"
```

Comment:
Add the option to the library, not to the util, otherwise the option will only be visible under the library package if you select the utility.
Instead of opening a new menu, which adds one line to the Compression menu, use that line for the option itself instead, avoiding the need to open a new menu, by removing the `menu` line and adding the `depends` to the `config` definition:
```
config LZ4_OPTIMIZE_SPEED
	bool "Optimize for speed"
	depends on PACKAGE_liblz4
	help
		This enables additional optimization and
		increases performance considerably at
		the expense of binary size.

endif
```
will turn
```
<*> liblz4........................................ Extremely fast compression
      Configuration  --->
```
into
```
<*> liblz4........................................ Extremely fast compression
[ ]   Optimize for speed (NEW)
```
If you look at the current menu, notice that libzstd's option is not done right.

Reply:
I like this as well. Fixed.

I will probably fix libzstd when I get the chance.